counterpartylib/test/complex_unit_test.py

```diff
@@ -0,0 +1,141 @@
+import pprint
+import tempfile
+import pytest
+from counterpartylib.test import conftest  # this is require near the top to do setup of the test suite
+from counterpartylib.test import util_test
+from counterpartylib.test.util_test import CURR_DIR
+from counterpartylib.test.fixtures.params import DP, ADDR
+
+from counterpartylib.lib import util
+
+FIXTURE_SQL_FILE = CURR_DIR + '/fixtures/scenarios/unittest_fixture.sql'
+FIXTURE_DB = tempfile.gettempdir() + '/fixtures.unittest_fixture.db'
+
+
+@pytest.mark.usefixtures("server_db")
+@pytest.mark.usefixtures("api_server")
+def test_alice_bob(server_db):
+    alice = ADDR[0]
+    bob = "miJqNkHhC5xsB61gsiSWXeTLnEGSQnWbXB"
+
+    # check alices UTXOs
+    utxos = util.api('get_unspent_txouts', {"address": alice})
+    assert len(utxos) == 1
+    assert utxos[0]['address'] == alice
+    assert utxos[0]['txid'] == "ae241be7be83ebb14902757ad94854f787d9730fc553d6f695346c9375c0d8c1"
+    assert utxos[0]['amount'] == 1.9990914
+    assert utxos[0]['confirmations'] == 74
+
+    # balance before send
+    alice_balance = util.get_balance(server_db, alice, 'XCP')
+    bob_balance = util.get_balance(server_db, bob, 'XCP')
+    assert alice_balance == 91950000000
+    assert bob_balance == 0
+
+    # create send
+    v = int(100 * 1e8)
+    send1hex = util.api('create_send', {'source': alice, 'destination': bob, 'asset': 'XCP', 'quantity': v})
```

Comment:
do you see any potential reliability issues using util.api vs adding a more direct method (i.e. direct to code vs JSON RPC request over localhost)?  I don't really, but it may be a bit quicker/cleaner.